tests/ha/setup.sh

```diff
@@ -1,14 +1,22 @@
-set -xe
```

Comment:
As discussed, please restore tests/ha/setup.sh, in order to preserve the new code, create a new setup file please.